.github/workflows/ci.yaml

```diff
@@ -164,6 +164,20 @@ jobs:
           api-level: ${{ matrix.api-level }}
           script: ./gradlew connectedAndroidTest
 
+  android-delete-artifacts:
+    needs: android-test
+    if: always()
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Delete artifacts
+        uses: geekyeggo/delete-artifact@v1
+        with:
+          name: |
+            x86
+            x86_64
+            arm64-v8a
```

Comment:
It'd be good to add a comment here explaining that this list should be kept in sync with the matrix build in `android-build`, and a comment there pointing back to here.

Reply:
Good call. Done.
The action I'm using is planning on adding wildcard support or a "delete all" flag, so this will become more easily maintained once that goes through.